client/src/main/java/tds/assessment/SegmentItemInformation.java

```diff
@@ -47,10 +47,10 @@ public Segment getSegment() {
     }
 
     /**
-     * @return the parent assessment's {@link tds.assessment.Item}.  If a single segmented assessment this will be empty
+     * @return the assessment's {@link tds.assessment.Item}.  If a single segmented assessment this will be empty
      */
-    public List<Item> getParentItems() {
-        return parentItems;
+    public List<Item> getSiblingItems() {
```

Comment:
This was a misunderstanding on my part.  This needs to contain any item that isn't part of the current segment.  I mistakenly thought it was the parent items which never really happens in a multi segmented assessment.